publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -182,6 +198,19 @@ public PublishPublicationStatusResponse publishPublication(UserInstance userInst
         return updateResourceService.publishPublication(userInstance, resourceIdentifier);
     }
 
+    public Publication autoImportPublication(Publication inputData) {
```

Comment:
Not sure if it is better to take in ImportCandidate as variable, so none in future uses this method to create publication.

Reply:
Yes, do that.